webpack/assets/javascripts/react_app/components/ReportTemplatesTable/index.js

```diff
@@ -0,0 +1,12 @@
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import ReportTemplatesTable from './ReportTemplatesTable';
+import reducer from './ReportTemplatesTableReducer';
+import * as actions from './ReportTemplatesTableActions';
+
+const mapStateToProps = state => state.report_templates_table;
+const mapDispatchToProps = dispatch => bindActionCreators(actions, dispatch);
+
+export const reducers = { report_templates_table: reducer };
+
+export default connect(mapStateToProps, mapDispatchToProps)(ReportTemplatesTable);
```

Comment:
We are moving to the new hooks approach which is just simpler to read and understand.
can you please use `useDipsatch` and `useSelector` instead of this `connect` HOC?
There are some components that use this approach: ConnectedTypeAheadSelect, FactCharts, ForemanModal and more for reference